zulipterminal/cli/run.py

```diff
@@ -352,7 +369,20 @@ def main(options: Optional[List[str]]=None) -> None:
         else:
             theme_data = THEMES[theme_to_use[0]]
 
+        ZULIPRC_CONFIG = 'in zuliprc file'
+
+        if zterm['footlinks'][1] == ZULIPRC_CONFIG:
+            if zterm['footlinks'][0] == 'disabled':
+                maximum_footlinks = 0
+            else:
+                # Random Large Figure which practically
+                # wouldn't be there in the messages
+                maximum_footlinks = 1000
```

Comment:
This should be 3; see my general feedback.

Reply:
Ahem. After running with the non-PR dev version quite a bit, I see lots of footlinks in some cases, so 1000 seems like a reasonable limit after all. Ideally we wouldn't use a fixed high number but something more programmatic, eg rather than comparing x to 1000 in a conditional we could wrap it and have a bool be True or False in this case, and dependent on the number otherwise.